core_backend/app/dashboard/models.py

```diff
@@ -171,7 +174,8 @@ def get_time_labels_query(
         )
         .select_from(
             text(
-                f"generate_series('{start_date}', '{end_date}', '1 {interval_str}'"
+                f"generate_series('{start_date}'::timestamp, '{end_date}'::timestamp + "
+                f"'1 day'::interval, '1 {interval_str}'"
```

Comment:
This is to fix a bug where the most recent records don't get picked up. E.g. if we ask for 1 day increments for Jan 1st PM to Jan 5th 5PM, it will miss out of record created between Jan 5th 12AM and Jan 5th 5PM. 

This adds an extra interval to fix this issue